circle.yml

```diff
@@ -31,6 +31,11 @@ dependencies:
     - /usr/local/android-sdk-linux/tools
     - /usr/local/android-sdk-linux/build-tools/25.0.1
 
+test:
+  override:
+    - ./gradlew assembleRelease --stacktrace -Pcom.android.build.threadPoolSize=1 -Dorg.gradle.parallel=false -Dorg.gradle.jvmargs="-Xms512m -Xmx2048m" -Dorg.gradle.daemon=false
```

Comment:
That `GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'` line in the script is the way CircleCI docs recommends to deal with the memory issues, I don't think we need this here, do we?